packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -73,6 +73,9 @@ module.exports = {
     // enforces consistent naming when capturing the current execution context
     'consistent-this': 'off',
 
+    // enforce that default parameters should come last
+    'default-param-last': ['error'],
```

Comment:
please update the [guide section](https://github.com/airbnb/javascript#functions--defaults-last) to reference the eslint rule.

This change is semver-minor, since it's enforcing something already in the guide.

Reply:
Hello @ljharb: are you referring to something in addition to the change in [README.md](https://github.com/airbnb/javascript/pull/2106/files#diff-04c6e90faac2675aa89e2176d2eec7d8) I made?

Reply:
nope, that covers it! (sorry if i missed it the first time)